### PR TITLE
Validator for pyflaking the whole repo + renamed a variable in JobQueue

### PR DESCRIPTION
**Pull request by icanberk**
Now we can do:
`python tests/validate.py`

I also changed `jobQueue.jobQueue` to `jobQueue.liveJobs` because it was too confusing. Let me know if you have a better name suggestions. I also ran the tests after this change.

### CODE REVIEW

**Comment by icanberk** [comment]
Oh and I fixed the pyflake warnings. Mostly unnecessary dependencies and couple obvious mistakes.

**Comment by dlbucci** [comment]
Is this basically rubocop for python, or is there less of a focus on style?

**Comment by icanberk** [comment]
It's not that intense about style, but I think it's really good at static analysis part.

File: restful-tango/tangoREST.py [hunk 1]
```diff
@@ -180,7 +180,7 @@ def convertJobObj(self, dirName, jobObj):
                 input = input,
                 timeout = timeout,
                 notifyURL = notifyURL,
-                maxOutputFileSize = 4096)
+                maxOutputFileSize = maxOutputFileSize)
```
Comment (mihirpandya) [inline restful-tango/tangoREST.py]: This should be `Config.MAX_OUTPUT_FILE_SIZE`

**Comment by mihirpandya** [comment]
Has it caught anything yet? Also, this repo have a continuous integration system to, just like the Autolab repo. How can we get started..?

File: vmms/ec2SSH.py [hunk 2]
```diff
@@ -259,8 +259,8 @@ def runJob(self, vm, runTimeout, maxOutputFileSize):
                 %d -o %d autolab &> output" % (
                         config.Config.VM_ULIMIT_USER_PROC, config.Config.VM_ULIMIT_FILE_SIZE,
                         runTimeout, maxOutputFileSize) 
-                return timeout(["ssh"] + Ec2SSH._SSH_FLAGS +
```
Comment (icanberk) [inline vmms/ec2SSH.py]: It caught this indentation error.

File: restful-tango/tangoREST.py (hunk 1 above)
Comment (icanberk) [inline restful-tango/tangoREST.py]: So
```
if 'max_kb' in jobObj:
             maxOutputFileSize = jobObj['max_kb']
```

Is deletable?

**Comment by icanberk** [comment]
Consult with @ymzong for CI.

**Comment by icanberk** [comment]
@mihirpandya this good to go?

**Comment by icanberk** [comment]
This fixes #17

**Comment by icanberk** [comment]
ping @mihirpandya

**Comment by mihirpandya** [comment]
Apologies. Will look at this tonight.

**Comment by mihirpandya** [comment]
Looks good. Could run job1 and job2.

**Comment by icanberk** [comment]
@dlbucci would be proud

**Pull request merged**